docs/developer/decision-records/2022-04-19-jwt-hardening/README.md

```diff
@@ -0,0 +1,34 @@
+# JWT hardening
```

Comment:
nit: Do we also want to include expiration claim? https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.4

Reply:
This one is already set in EDC to 10 mins 👍🏻